modules/chemical_reactions/doc/content/source/actions/ChemicalCompositionAction.md

```diff
@@ -7,50 +7,58 @@ This action is designed for use with the thermochemistry library Thermochimica [
 
 The `ChemicalComposition` action is used to initiate the framework for thermochemical calculations using the thermodynamics solver [_Thermochimica_](https://github.com/ORNL-CEES/thermochimica). The action creates variables needed for the analysis, reads the thermodynamic model for material system, and sets the units for temperature, pressure, and elemental amounts. It can optionally initiate the amounts of elements used in the model.
 
-The `ChemicalComposition` action is also used to specify lists of chemical phases and chemical species for which concentrations should be output at each timestep. The phase and species names must match those in the data file specified. The `ChemicalComposition` action creates all AuxVariables and AuxKernels needed for output of phase concentration data. The list of phases is specified in the `output_phases` variable, and the list of species is specified in `output_species`. The species must be specified in the convention `phase_name:species_name` so that the desired species in the desired phase is uniquely identified. A combination of an AuxKernel derived from `ThermochimicaAux` and a UserObject derived from `ThermochimicaNodalData` can then be used to store the concentrations of these species in their respective phases.
+The `ChemicalComposition` action is also used to specify lists of chemical phases and chemical species for which concentrations should be output at each timestep. The phase and species names must match those in the data file specified. The `ChemicalComposition` action creates all AuxVariables needed for output of phase concentration data. The list of phases is specified in the `output_phases` variable, and the list of species is specified in `output_species`. The species must be specified in the convention `phase_name:species_name` so that the desired species in the desired phase is uniquely identified. Alternatively, users can specify `ALL` to output all species. Other desired outputs such as element potentials, vapor pressures of species in gas phase and moles of elements in specified phases can be specified using `output_element_potentials`, `output_vapor_pressures`, and `output_element_phases` respectively. The `ChemicalComposition` action creates a UserObject derived from `ThermochimicaNodalData` which runs Thermochimica at each node and stores the output variables.
```

Comment:
```suggestion
The `ChemicalComposition` action is also used to specify lists of chemical phases and chemical species for which concentrations should be output at each timestep. The phase and species names must match those in the data file specified. The `ChemicalComposition` action creates all AuxVariables needed for output of phase concentration data, chemical potentials, vapor pressures, etc.. The list of phases is specified in the `output_phases` variable, and the list of species is specified in `output_species`. The species must be specified in the convention `phase_name:species_name` so that the desired species in the desired phase is uniquely identified. Alternatively, users can specify `ALL` to output all species. Other desired outputs such as element potentials, vapor pressures of species in gas phase and moles of elements in specified phases can be specified using `output_element_potentials`, `output_vapor_pressures`, and `output_element_phases` respectively. The `ChemicalComposition` action creates a UserObject derived from `ThermochimicaNodalData` which runs Thermochimica at each node and stores the output in the created auxvariables.
```